src/components/DataCollector/index.js

```diff
@@ -51,7 +52,8 @@ createDataCollector.namespace = "DataCollector";
 
 createDataCollector.configValidators = {
   collectionUrl: {
-    defaultValue: "https://edgegateway.azurewebsites.net"
+    defaultValue: "https://edgegateway.azurewebsites.net",
+    validate: [required, matchesRegEx("^https://[^s/$.?#].[^s]*$")]
```

Comment:
Why do you have required here?  It isn't required because there is a default value.

Reply:
I'm not sure if we will have a default, but I can remove the required for now